pkg/operator/ceph/csi/spec.go

```diff
@@ -498,6 +522,23 @@ func deleteCSIDriverResources(
 	return succeeded
 }
 
+func applyCephClusterNetworkConfig(objectMeta *metav1.ObjectMeta, rookclientset rookclient.Interface) (bool, error) {
+	cephClusters, err := rookclientset.CephV1().CephClusters(objectMeta.Namespace).List(metav1.ListOptions{})
+	if err != nil {
+		return false, errors.Errorf("failed to find CephClusters in namespace %q", objectMeta.Namespace)
+	}
+	for _, cephCluster := range cephClusters.Items {
+		if cephCluster.Spec.Network.IsMultus() {
+			err = k8sutil.ApplyMultus(cephCluster.Spec.Network.NetworkSpec, objectMeta)
+			if err != nil {
+				return false, errors.Wrapf(err, "failed to apply multus configuration to CephCluster %q", cephCluster.Name)
+			}
+		}
+	}
+
+	return true, nil
```

Comment:
even if the ceph cluster is not using multus it's returning true? 
don't we need to return false if the ceph cluster network is not multus?

Reply:
Good point, we should only return if multus got configured, I'm sending a PR to fix this.

Reply:
https://github.com/rook/rook/pull/6152